### PR TITLE
[DPEDE-545] Implement Footer component in UxPin

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-export const CHI_VERSION = '5.37.0';
+export const CHI_VERSION = '5.39.0';
```
Comment (josecarlosgonzalezv) [inline src/constants/constants.js:1]:
```suggestion
export const CHI_VERSION = '5.40.0';
```

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    'English',
+    'Español',
+    'Português',
+    'Français',
+    'Deutsch',
+    '简体中文',
+    '日本語',
+  ];
+  const footerLanguageClass = isMobile
+    ? ['-justify-content--center -w--100 -pr--0 -pb--2']
```
Comment (josecarlosgonzalezv) [inline src/components/Footer/Footer.js:21]: We can use the constants from the **classes.js**: `UTILITY_CLASSES.WIDTH[100]`

File: src/components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  render() {
+    const { id } = this.state;
+    const buttonId = `${id}-language-dropdown-button`;
+    const isTabletSize = this.props.footerSize === 'Portrait tablet';
+    const isDesktopSize = this.props.footerSize === 'Desktop';
+    const footerStyle = isDesktopSize
+      ? { minWidth: '1256px' }
+      : { width: isTabletSize ? '736px' : '368px' };
+    const footerInternalContentStyle = isDesktopSize ? '-flex--row' : '-flex--column';
+    const ulElementStyle = isDesktopSize ? '-flex--row' : '-flex--column';
```
Comment (josecarlosgonzalezv) [inline src/components/Footer/Footer.js:79]: We can use only one constant for both cases as they have the same value. Include `-flex--row` and column in the **classes.js** and use it here.

```suggestion
    const footerInternalContentStyle = isDesktopSize ? '-flex--row' : '-flex--column';
    const ulElementStyle = isDesktopSize ? '-flex--row' : '-flex--column';
```

File: src/components/Footer/Footer.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      { url: 'https://www.lumen.com/en-us/about/legal/commission-for-complaints-for-telecom-television-services-ccts', text: 'Canadian Ombudsman', newTab: false },
+      { url: 'https://assets.lumen.com/is/content/Lumen/gender-pay-gap-report?Creativeid=b565bf28-6d55-4f86-a245-2f0eb40b91a3', text: 'Gender Pay Gap 2020 (UK)', newTab: false },
+      { url: 'https://www.lumen.com/es-ar/about/legal/latam-tariffs-regulatory.html', text: 'Legal (Latin America)', newTab: false },
+      { url: 'https://www.lumen.com/en-us/about/legal/de-legal-notices.html', text: 'Legal Notices (Germany)', newTab: false },
+      { url: 'https://assets.lumen.com/is/content/Lumen/modern-slavery-statement-1?Creativeid=5dbb9687-e4a8-4a40-ae00-673ce51d6a80', text: 'Modern Slavery Statement 2020 (UK)', newTab: false },
+      { url: 'https://www.lumen.com/en-us/about/legal/uk-tax-strategy.html', text: 'UK Tax Strategy', newTab: false },
+    ];
+
+    return (
+      <>
```
Comment (josecarlosgonzalezv) [inline src/components/Footer/Footer.js:100]: Why do we need `<>` if there is only one element inside?

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.js (hunk 4 above)
Comment (soujanreddy) [reply src/components/Footer/Footer.js:100]: Removed

**Review by jllr — APPROVED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

File: src/components/Footer/Footer.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        aria-label="Select your preferred language">
+        <div className={BUTTON_CLASSES.CONTENT}>
+          <i
+            className={`${ICON_CLASS} icon-globe-network-outline`}
+            aria-hidden="true">
+          </i>
+          <span>English</span>
+        </div>
+      </a>
+      <div className={`${DROPDOWN_CLASSES.MENU} -w--sm ${UTILITY_CLASSES.TYPOGRAPHY.TEXT_BODY}`}>
```
Comment (jllr) [inline src/components/Footer/Footer.js:40]:
```suggestion
      <div className={`${DROPDOWN_CLASSES.MENU}  ${UTILITY_CLASSES.TYPOGRAPHY.TEXT_BODY}`}>
```

**Pull request merged**